src/app/src/main/kotlin/org/imperial/mrc/hint/security/tokens/OneTimeTokenAuthenticator.kt

```diff
@@ -18,17 +20,22 @@ class OneTimeTokenAuthenticator(
 
     private val tokenIssuer = appProperties.tokenIssuer
 
-    override fun createJwtProfile(credentials: TokenCredentials, jwt: JWT)
+    override fun createJwtProfile(
+        credentials: TokenCredentials?,
```

Comment:
Although these are technically nullable because they are Java properties, I would simply leave them as non-nullable in the Kotlin, since they should never be null in practice.